AndroidSurveySDK/.gitignore

```diff
@@ -0,0 +1,7 @@
+build
+.gradle
+.idea
+/build
+/.gradle
+/.idea
+/local.properties
```

Comment:
Can you double check the configuration of this `.gitignore` file? Because I see the `.gradle` and `.idea` are pushed on repo?

 
![image](https://user-images.githubusercontent.com/6950766/132729112-38071a41-cef3-404b-9477-abd7a9b14979.png)

Reply:
@hoangnguyen92dn Yes, they are pushed before I updated the .gitignore. I removed them already on the update.